integration-tests/jail/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  chain0:
+    build: .
+    volumes:
+      - ../..:/root/chain_src
+      - ./disk:/root/disk
+    image: chain_test0
+    ports:
+       - "1022:22"
+       - "9981:9981"
+       - "26656:26656"     
+       - "26657:26657"     
```

Comment:
as these ports are hardcoded, it won't be possible to run two tests in parallel -- @calvinaco @calvinlauco had something like that sorted in the existing integration tests

Reply:
@leejw51 @leejw51crypto Yes, there's a script to find a free port to use: https://github.com/crypto-com/chain/blob/master/ci-scripts/find-free-port.sh

Reply:
only vm0 needs external port-mapping for client-rpc, i'll remove un-necessary ports.